spec/helpers/service_helper_spec.rb

```diff
@@ -131,6 +133,33 @@
     end
   end
 
+  describe '#previous_service_url' do
+    let(:commodity_code) { '0702000007' }
+    let(:user_session) { build(:user_session) }
+
+    context 'when redirect_to is set' do
+      let(:user_session) { build(:user_session, redirect_to: 'https://example.com/chieg') }
+
+      it { expect(helper.previous_service_url(commodity_code)).to eq('https://example.com/chieg') }
+    end
+
+    context 'when TRADE_TARIFF_FRONTEND_URL is set' do
+      it 'returns the dev trade tariff tools url' do
```

Comment:
it is not clear from the test that it is actually set

Reply:
I agree with this. I'll update it in both places that we validate this